tools/msvc/sbclang.targets

```diff
@@ -711,13 +727,6 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="ComputeRuntimeInputs" DependsOnTargets="ComputeClangOptions;ComputeStarboardRequiresObjectiveCRuntime">
-    <ItemGroup>
-      <Link Condition="'$(StarboardLinkObjCRuntime)' == 'true' and '$(StarboardRequiresObjectiveCRuntime)' == 'true'" Include="$(StarboardObjCRuntimeLib)"/>
```

Comment:
what here ensures that libobjc2 is linked?

also, i'm worried about the libobjc2 build itself perhaps trying to links libobjc2 since we got rid of StarboadLinkObjcRuntime